packages/common/utils/src/get-set.spec.ts

```diff
@@ -17,6 +17,12 @@ describe('get function', () => {
   it('should work correctly with empty objects', () => {
     expect(get({}, 'a')).toBeUndefined();
   });
+  it('should handle paths leading to undefined or null values inside nested objects', () => {
```

Comment:
How about separate test cases which null and undefined?

I think it is better each test suite have one test case

Reply:
That's a good opinion. I separated the test cases into two describes.